src/Aws/General.hs

```diff
@@ -204,12 +207,26 @@ parseRegion =
     <|> UsEast1 <$ P.text "us-east-1"
     <|> UsWest1 <$ P.text "us-west-1"
     <|> UsWest2 <$ P.text "us-west-2"
+    <|> CustomEndpoint . T.pack <$> (P.text "custom:" *> many P.anyChar)
```

Comment:
I think there should be a Haddock comment for this function, since it is exported. We may also add a Haddock comment to the constructor itself.

Let me know if you want me to add the comments, such that you are not hold responsible for or are associated with this hack :-)